alpha-api/src/main/java/at/ac/tuwien/kr/alpha/api/terms/IntervalTerm.java

```diff
@@ -0,0 +1,14 @@
+package at.ac.tuwien.kr.alpha.api.terms;
+
+/**
+ * A term representing an interval within the integers.
+ * Intervals are syntactic sugar, e.g. the rule "p(X) :- X = 1..2." is a shorthand for
+ * "p(X) :- X = 1. p(X) :- X = 2.".
+ */
+public interface IntervalTerm extends Term{
```

Comment:
I believe that the code style requires a space before `{`. Maybe try formatting.

Reply:
Fixed, thanks!